locale/core.yml

```diff
@@ -320,6 +320,11 @@ core:
       username_placeholder: => core.ref.username
       welcome_text: "Welcome, {username}!"
 
+    # These translations are used to show user an alert if the account didn't verfied
```

Comment:
The comment is there to help localizers find the strings in the UI, so it would be a good idea to explain the conditions under which the message box is displayed. For example:

# These translations are used in a message box displayed when an unverified user _attempts to post a message_.

I'm not sure what the actual conditions are, so please change the italicized part as necessary.